source/overloading.tex

```diff
@@ -3440,331 +3442,295 @@
 \end{note}
 
 \pnum
+In the remainder of this section, \cvqual{vq} represents either
+\tcode{volatile} or the absence of \tcode{volatile}.
```

Comment:
Maybe "or no cv qualifier"? Because currently it sounds like you could use "vq cv" to take an arbitrary cv-qualifier and constrain it to not include `volatile`.